.github/workflows/releaseBuild.yaml

```diff
@@ -0,0 +1,127 @@
+# ReleaseBuild Workflow
+#
+# This GitHub Actions workflow performs the publication process for a FHIR IG Release.
+#
+# Based on these workflows:
+# https://github.com/medigree/base/blob/main/.github/workflows/ghbuild.yml
+# https://github.com/WorldHealthOrganization/smart-html/blob/main/.github/workflows/release.yml
+
+name: "ReleaseBuild"
+
+# Controls when the action will run.
+on:
+  workflow_call: # Reusable by other workflows.
+  # Triggers the workflow on push or pull request events.
+  push:
+    branches: 
+      # Only triggers on pushes on these branches.
+      - "master"
+      #- "releases/**"
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: "ubuntu-latest"
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Get branch name
+        run: echo ${GITHUB_REF##*/}
+      
+        # Cache Fish output for usage in other jobs.
+      - name: Output cache
+        id: fsh-cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            static/**
+            output/**
+            sushi-config.yaml
+          key: ${{ runner.os }}-fsh-${{ hashFiles('static/**') }}-${{ hashFiles('output/**') }}
+          restore-keys: ${{ runner.os }}-fsh-
+
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v4
+
+      - name: Update the image to the latest publisher
+        uses: docker://hl7fhir/ig-publisher-base:latest
```

Comment:
I think we may have previously used a fixed version of the publisher to avoid things from breaking if an error-prone publisher is released? Have you talked to @jvitrifork about it?

Reply:
I think we should stay on `latest`. We can always revert to a fixed version if need be.

Reply:
Yeah. I agree. I can't think of any emergencies where we would need the pipeline to run. If it doesn't work the bucket would just not get updated, and if we are in need to temporarily pin it e.g. before a release, it is easy to do so.